aggregator/src/test/java/org/jenkinsci/plugins/workflow/steps/scm/MercurialStepTest.java

```diff
@@ -90,7 +90,7 @@ private static void hg(File repo, String... cmds) throws Exception {
             "        }\n" +
             "        dir('other') {\n" +
             "            checkout([$class: 'MercurialSCM', source: $/" + otherRepoU + "/$, clean: true])\n" +
-            "            try {\n" +
+            "            try {\n" + // TODO or use fileExists
```

Comment:
BTW `WaitForConditionStepTest.catchErrors` also uses this idiom but catching `FileNotFoundException` there is useful for test coverage purposes as explained in a comment there.